src/components/LanguageSwitch/LanguageSwitch.module.css

```diff
@@ -23,15 +23,17 @@
 }
 
 .LanguageSwitchCurrentIcon {
-  margin-left: 5px;
+  position: relative;
+  top: 1px;
+  margin-left: 10px;
   color: var(--txt-1);
 }
 
 .LanguageSwitchList {
   position: absolute;
   top: 20px;
   z-index: 99;
-  right: 21px;
+  right: 0px;
```

Comment:
![image](https://user-images.githubusercontent.com/7966133/153504582-461e81ce-6761-4d68-9c4b-f0f8f9e3f984.png)

It's should be `25px`


![image](https://user-images.githubusercontent.com/7966133/153504736-91207009-56b3-41b6-8e28-a8ea884eb375.png)

Reply:
Yeah, that looks great! 🚀

Reply:
Or we can just use `left: 0px` instead of `right` value!